src/dotnet/ToolManifest/IToolManifestEditor.cs

```diff
@@ -9,5 +9,6 @@ internal interface IToolManifestEditor
     {
         void Add(FilePath to, PackageId packageId, NuGetVersion nuGetVersion, ToolCommandName[] toolCommandNames);
         void Remove(FilePath fromFilePath, PackageId packageId);
+        void Edit(FilePath to, PackageId packageId, NuGetVersion newNuGetVersion, ToolCommandName[] newToolCommandNames);
```

Comment:
Is `to` the path to the manifest?  Perhaps a more descriptive name of `manifest`?  Same goes for `Add` and perhaps `Remove`.

Reply:
Used manifest